src/bindings/python/tests/test_graph/test_core.py

```diff
@@ -392,6 +392,23 @@ def test_discrete_type_info():
     assert n3.get_type_info().name < n1.get_type_info().name
     assert n3.get_type_info().name <= n1.get_type_info().name
 
+@pytest.mark.parametrize("input_data", [
+    [1, 2, 3],
+    (1, 2, 3),
+])
```

Comment:
```suggestion
])
@pytest.mark.parametrize("shape_type", [Shape, PartialShape])
```
I'd also prefer to keep one test (as they are the same) and parametrize shape type like this.